frontend/src/index.js

```diff
@@ -25,6 +26,11 @@ const router = createBrowserRouter([
         element: <ExplorePage />,
         errorElement: <ErrorPage />,
       },
+      {
+        path: 'create',
+        element: <CreatePage />,
```

Comment:
Same here, maybe change the component name to be like `AICreationPage` or `AICreatePage`.